internal/pkg/storage/db/storage.go

```diff
@@ -233,6 +231,33 @@ func (s *Storage) Count() int {
 	return count
 }
 
+// GetUniqueUsers return slice of unique user ID
+func (s *Storage) GetUniqueUsers(ctx context.Context) []int {
```

Comment:
```suggestion
func (s *Storage) GetUniqueUsers(ctx context.Context) (int64, error) {
```
Если записей будет хотя бы сотня тысяч, начнутся заметные тормоза. И не забываем про ошибки выполнения.

Reply:
`userID` у меня `int`, `len()` тоже возвращает `int`, соответственно кол-во уникальных юзеров будет не больше `int` и по факту равно текущему максимальному значению в `userID`